source/_patterns/03-templates/content-grid/content-grid.twig

```diff
@@ -0,0 +1,46 @@
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['content-grid']) }) -%}
+
+{%- set hasSecondary = false -%}
+{%- for area in content if area.area|default('full') == 'secondary' -%}
+  {%- set hasSecondary = true -%}
+{%- endfor -%}
+
+{%- if hasSecondary -%}
+  {%- set attributes = attributes|merge({class: attributes.class|merge(['content-grid--has-secondary']) }) -%}
+{%- endif -%}
+
+<{{ element|default('div') }} {%- include 'atoms-attributes' %}>
+
+  {%- for area in content -%}
+    {%- with area only -%}
+
+      {%- block item -%}
+
+        {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['content-grid__item']) }) -%}
+
+        {%- if area is defined -%}
+
+          {%- set attributes = attributes|merge({class: attributes.class|merge(['content-grid__item--' ~ area]) }) -%}
+
+          <div {%- include 'atoms-attributes' %}>
+            {%- include 'atoms-text' with {nodes: content} -%}
+          </div>
+
+        {%- else -%}
+
+          {%- with arguments|merge({_template: template}) only -%}
+
+            {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['content-grid__item'])}) -%}
```

Comment:
Default of `main`, otherwise the pattern itself can set its area: https://github.com/libero/pattern-library/blob/abc73988082e5b37dcfd40d24a82c76356188af8/source/_patterns/02-organisms/content-header/content-header.twig#L3-L6